src/components/button/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledButton, StyledLink } from './styles';
+
+const Button = ({ label, href, size }) => {
+  if (href) {
```

Comment:
You don't need this condition here in order to distinguish between a button and a link. Styled-components gives you the [as](https://styled-components.com/docs/api#as-polymorphic-prop) prop which you can use for the button in the HeroSection to change it to a Link

This way you can delete this component and use the styles file directly.

Reply:
should I remove the button component? and keep button styles in hero.styles.js?

Reply:
You can keep the button/styles.js and export it from button/index.js 

There's no need for the button/Button.js component anymore I guess

Reply:
okay